extras/shell/src/main/java/org/apache/rya/shell/RyaAdminCommands.java

```diff
@@ -141,7 +174,8 @@ public boolean arePCJCommandsAvailable() {
         // The PCJ commands are only available if the Shell is connected to an instance of Rya
         // that is new enough to use the RyaDetailsRepository and is configured to maintain PCJs.
         final ShellState shellState = state.getShellState();
-        if(shellState.getConnectionState() == ConnectionState.CONNECTED_TO_INSTANCE) {
+        if(shellState.getConnectionState() == ConnectionState.CONNECTED_TO_INSTANCE &&
+                shellState.getStorageType().get() == StorageType.ACCUMULO) {
```

Comment:
Is this review supposed to include shell updates? Because if so, that work does not appear to be done yet. I'd encourage that work being a separate ticket.

Reply:
I'm not too concerned with a lot of those since this PR is the last in the expected merge order.  If it causes any problems, it'll show up with a merge conflict and I'll take care of it then

Reply:
Check RyaAdminCommandsTest in the rya.shell project.